benchmark/benchmark_utils.hpp

```diff
@@ -84,8 +84,66 @@ inline auto get_random_data(size_t size, T min, T max, size_t max_random_size =
     return data;
 }
 
+#if defined(_WIN32) && defined(__clang__)
+// get_random_data() generates only part of sequence and replicates it,
+// because benchmarks usually do not need "true" random sequence.
+template<>
+inline std::vector<unsigned char> get_random_data(size_t size, unsigned char min, unsigned char max, size_t max_random_size)
+{
+    std::random_device rd;
+    std::default_random_engine gen(rd());
+    std::uniform_int_distribution<int> distribution(min, max);
+    std::vector<unsigned char> data(size);
```

Comment:
Maybe better to keep using the function with template parameters and just insert:

```suggestion
    using distribution_type = std::conditional<(sizeof(T)==1), short, T>::type;
    std::uniform_int_distribution<distribution_type> distribution(min, max);
```

Reply:
Thanks for the suggestion - implemented in the PR now.